### PR TITLE
Stop installing terryfy.

### PR DESCRIPTION
**Pull request by jamadden**

### CODE REVIEW

File: MANIFEST.in [hunk 1]
```diff
@@ -1,3 +1,4 @@
+pruny terryfy
```
Comment (mgedmin) [inline MANIFEST.in]: Typo -- I think you meant `prune`.

File: setup.py [hunk 2]
```diff
@@ -173,7 +173,9 @@ def BTreeExtension(family):
       url="https://github.com/zopefoundation/BTrees",
       license="ZPL 2.1",
       platforms=["any"],
-      packages=find_packages(),
+      # Make sure we don't get 'terryfy' included in wheels
+      # created on macOS CI
+      packages=find_packages(include=("BTrees",)),
```
Comment (mgedmin) [inline setup.py]: This is BTW a yet another reason to prefer a `src` subdirectory at the project root  ;)

File: MANIFEST.in (hunk 1 above)
Comment (jamadden) [reply MANIFEST.in]: Yikes, how'd that happen. Thanks.

Comment (jamadden) [reply MANIFEST.in]: `python setup.py sdist` outputs a warning:
```
reading manifest template 'MANIFEST.in'
warning: manifest_maker: MANIFEST.in, line 1: unknown action 'pruny'
```

But `check-manifest -v` does not.

Comment (mgedmin) [reply MANIFEST.in]: Yeah, it would be good if check-manifest would syntactic validity as well ->https://github.com/mgedmin/check-manifest/issues/85

**Pull request merged**